### PR TITLE
use tabbed interface for better ux

### PR DESCRIPTION
**Pull request by rsisson**

### CODE REVIEW

**Comment by jacobsfletch** [comment]
Hey @rsisson thanks for the contribution! I wonder if we should make this _opt-in_ via the plugin's config. I could imagine some users _wanting_ see all fields at the same time.

**Comment by rsisson** [comment]
Something like this latest commit?

File: src/types.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export type SEOConfig = {
   globals?: string[]
   uploadsCollection?: string
   fields?: Partial<Field>[]
+  tabbedUi?: boolean
```
Comment (jacobsfletch) [inline src/types.ts:8]: Change the casing of this to `tabbedUI`, then we're golden.

Comment (rsisson) [reply src/types.ts]: Done 👍

**Comment by jacobsfletch** [comment]
Thanks @rsisson I'll get this out in the next release, which could happen as early as today.

**Comment by jacobsfletch** [comment]
Hey @rsisson, a new version has been deployed. Run `yarn add @payloadcms/plugin-seo@^1.0.8" to give it a shot!

**Comment by rsisson** [comment]
@jacobsfletch - just updated to the latest version in my project and I'm not seeing *any* SEO fields now, regardless of `tabbedUI` value. No error messages as far as I can tell. Can you replicate?

**Comment by jacobsfletch** [comment]
@rsisson it's working in the local demo [here](https://github.com/payloadcms/plugin-seo/blob/main/demo/src/payload.config.ts#L49). This imports uncompiled source code, though, so it would be more accurate to import the dist or even spin up a fresh `create-payload-app` and install it there.

It is also possible to link the plugin to your project so you can develop on your instance. This can be done via Yarn link or a Webpack alias. At some point, I will add some information in the README for how to do this.

**Comment by rsisson** [comment]
@jacobsfletch seems to be working fine in prod, so it must be something with my local env. Thanks for taking a look!

**Pull request merged**